example/lib/main.dart

```diff
@@ -51,20 +55,23 @@ void main() {
 ///Initialize function is always called at the start of your application or within
 ///the scope of use of LexActivator
 void initializeLexActivator() {
-  LexActivator.SetProductData(productData: "PASTE_CONTENT_OF_PRODUCT.DAT_FILE");
+  LexActivator.SetProductData(productData: "SET_PRODUCT_DATA");
 
   LexActivator.SetProductId(
-      productId: "PASTE_PRODUCT_ID", flags: LexActivator.LA_USER);
+      productId: "SET_PRODUCT_ID", flags: LexActivator.LA_IN_MEMORY);
 
   LexActivator.SetAppVersion(appVersion: 'PASTE_YOUR_APP_VERSION');
 }
 
 ///Activation function is called once when license is to be activated. Calling this function
 ///again is not required.
 void activateLicense() {
-  LexActivator.SetLicenseKey(licenseKey: 'PASTE_LICENSE_KEY');
-  LexActivator.SetActivationMetadata(key: 'Metadata 1', value: 'Value 1');
+  LexActivator.SetLicenseKey(licenseKey: 'SET_LICENSE_KEY');
+  // LexActivator.SetActivationMetadata(key: 'Metadata 1', value: 'Value 1');
+  // LexActivator.SetActivationLeaseDuration(leaseDuration: 86400 * 60 * 2);
+  print('Activating license');
```

Comment:
@mumin-khan No need for this line: print('Activating license');

Reply:
@ahmad-kemsan done.